kvmagent/kvmagent/plugins/vm_config.py

```
@@ -94,6 +95,9 @@ def check_guest_tools_state(domain):
                 if not config:
                     logger.debug("read /usr/local/zstack/guesttools failed")
                     return qga_status
+                timeProtocol = qga.guest_exec_bash_no_exitcode(
+                    "grep -v '^[[:space:]]*#' /etc/chrony.conf | grep '/dev/ptp0' > /dev/null && echo 'ptp' || echo 'ntp'").strip()
+                qga_status.timeProtocol = timeProtocol
             except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)